gradle-talend-component/src/main/java/org/talend/sdk/component/gradle/TaCoKitPlugin.java

```diff
@@ -149,5 +150,16 @@ public void apply(final Project project) {
                 put("description", "Creates a Component ARchive (.car) based on current project.");
             }
         }, "talendComponentKitComponentArchive");
+
+        // deploy in studio
+        project.task(new HashMap<String, Object>() {
+
+            {
+                put(Task.TASK_TYPE, DeployInStudioTask.class);
+                put(Task.TASK_GROUP, "Talend Component Kit deployment");
+                put(Task.TASK_DESCRIPTION, "Deploys the module components to the Studio.");
+                put(Task.TASK_DEPENDS_ON, "jar");
+            }
+        }, "deployInStudio");
```

Comment:
Can you add the same prefix than other tasks please for consistency and get a better output when using tasks task from the cli?

Reply:
Update this one as well to use the same prefix and group. 👍